pkg/storage/v1/storage_nocache.go

```diff
@@ -70,12 +70,6 @@ func (sc StorageNoCache) GetSBOM(name string) (*v1beta1.SBOMSPDXv2p3, error) {
 		return nil, err
 	}
 
-	// remove fields that are not needed
```

Comment:
I will never again copy paste from old code during a refactoring... it's only good to forget things to cleanup later.